README.md

```diff
@@ -74,6 +74,9 @@ This project uses GitHub as a database. When you login, you will be presented wh
 
 This project uses the sw-precache to kickstart an offline cache. The offline cache only registers in production. If service needs to be manually removed make an **unregister** call from the registerServiceWorker.js import. 
 
+## 🍕Community
```

Comment:
Added Space, Markdown Standard for better readability, and reduce text congestion!

```suggestion
## 🍕Community

```